.github/issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!---
+ Please use this template to fill your issue, issues that does not
+ follow this will be asked to be edited to match the template, or will be closed
+
+ We use github issues for problem/bug reports, for questions, please use the Discord channel, the link for it can be found on the project README
```

Comment:
```suggestion
 We use Github issues for problem/bug reports; for questions, please use the Discord channel, the link for which can be found on the project README.
```

Reply:
Too many commas, split up in two sentences.